GetBPDCitizen/handler.ts

```diff
@@ -75,16 +76,25 @@ export const toApiBPDCitizen = (
 };
 
 export function GetBPDCitizenHandler(
-  citizenRepository: TaskEither<Error, Repository<Citizen>>
+  citizenRepository: TaskEither<Error, Repository<Citizen>>,
+  insertOrReplaceEntity: InsertOrReplaceEntity
 ): IHttpHandler {
   return async (context, _, requestFiscalCode) => {
+    const logRowData: AuditLogTableRow = {
+      AuthLevel: "Admin",
+      Citizen: requestFiscalCode.toUndefined(),
+      OperationName: "GetBPDCitizen",
+      PartitionKey: _.oid, // We can use email?
```

Comment:
why not? :)

Reply:
My only concern about that the email field is an array, we must take the first.